msbuild/ILMerge.targets

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <PackageReference Include="ILRepack" Version="2.0.18" />
+  </ItemGroup>
+
+  <Target Name="ILRepack" AfterTargets="CoreCompile" DependsOnTargets="CoreCompile" Inputs="@(IntermediateAssembly -&gt; '%(FullPath)')" Outputs="$(IntermediateOutputPath)ilrepack.txt" Returns="@(MergedAssemblies)" Condition="Exists(@(IntermediateAssembly -&gt; '%(FullPath)')) And '$(ILRepack)' != 'false'">
+    <ItemGroup>
+      <NetstandardPath Include="@(ReferencePath -&gt; '%(RootDir)%(Directory)')" Condition="'%(FileName)%(Extension)' == 'netstandard.dll'" />
+    </ItemGroup>
+    <PropertyGroup>
+      <NetstandardDirectory>@(NetstandardPath)</NetstandardDirectory>
+    </PropertyGroup>
+    <!-- The assemblies are in a very different place when building for netstandard2.0, which means we need different logic to find them too -->
+    <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
Should this be `$(TargetFramework.StartsWith('netstandard'))` instead to make it more future-proof?